ush/config.sh.RRFS_AK_dev2

```diff
@@ -0,0 +1,77 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+EXPT_BASEDIR="/misc/whome/rtrr/RRFS"
+EXPT_SUBDIR="RRFS_AK"
+
+QUEUE_DEFAULT="batch"
+QUEUE_HPSS="service"
+QUEUE_FCST="batch"
+
+VERBOSE="TRUE"
+
+RUN_ENVIR="nco"
+PREEXISTING_DIR_METHOD="rename"
+
+EMC_GRID_NAME="GSD_RRFSAK_3km" 
+GRID_GEN_METHOD="ESGgrid"
+
+QUILTING="TRUE"
+USE_CCPP="TRUE"
+CCPP_PHYS_SUITE="FV3_GSD_SAR"
+FCST_LEN_HRS="12"
+LBC_SPEC_INTVL_HRS="3"
+
+HALO_BLEND=10
+
+WTIME_RUN_FCST="02:30:00"
+
+DATE_FIRST_CYCL="20210101"
+DATE_LAST_CYCL="20211230"
+CYCL_HRS=( "00" "12" )
+
+WTIME_RUN_ANAL="00:25:00"
+DO_DACYCLE="TRUE"
+
+EXTRN_MDL_NAME_ICS="RAPX"
+EXTRN_MDL_NAME_LBCS="RAPX"
+
+envir="para"
+
+NET="RRFS_AK"
+
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_ak"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/home/rtrr/RRFS/graphics"
+NCL_REGION="alaska"
+MODEL="RRFS_AK (dev2)"
+
+#
+# In NCO mode, the following don't need to be explicitly set to "FALSE" 
+# in this configuration file because the experiment generation script
+# will do this (along with printing out an informational message).
+#
+#RUN_TASK_MAKE_GRID="FALSE"
+#RUN_TASK_MAKE_OROG="FALSE"
+#RUN_TASK_MAKE_SFC_CLIMO="FALSE"
+
+RUN="RRFS_AK"
+COMINgfs=""
+
+STMP="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2/stmp"  # Path to directory STMP that mostly contains input files.
+PTMP="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2/ptmp"  # Path to directory STMP that mostly contains input files.
```

Comment:
STMP and PTMP paths should point to RRFS space.  PTMP comment is also incorrect

Reply:
Done